components/Nav/Navbar.js

```diff
@@ -150,9 +150,10 @@ class ModalContainer extends PureComponent {
   }
 }
 
-const LogoLink = styled(Link).attrs((/* props */) => ({
+const LogoLink = styled(Link).attrs(({ ariaLabel }) => ({
```

Comment:
I think you can just pass `aria-label` directly on `LogoLink` instead of using the `attrs` indirection:

```tsx
<LogoLink aria-label="Styled Components Home Page">
```